docs/index.rst

```diff
@@ -97,10 +97,10 @@ greenlets. This can eliminate race conditions and greatly simplify the
 programming task.
 
 Also, threads require resources from the operating system (the thread
-stack, and bookeeping in the kernel). Because greenlets are
+stack, and bookkeeping in the kernel). Because greenlets are
 implemented entirely without involving the operating system, they can
 require fewer resources; it is often practical to have many more
-greenlets than it is threads.
+greenlets than the number of threads.
```

Comment:
```suggestion
greenlets than it is threads.
```

The original wording is correct; the new wording implies some relationship between threads and greenlets that doesn't exist.

Reply:
Although I must admit not understanding the sentence structure for the orignal sentence, English is but my second language, so I'll trust your grammar. Reverted the sentence.

Reply:
No worries! English is weird. 

In the phrase "it is often practical to have many more greenlets than it is threads," the word "it" stands in for the entire clause "it is often practical to have."  So the fully expanded version would be "it is often practical to have many more greenlets than it is [often] practical to have threads." Even expanded, the second "often" would usually be omitted as redundant or implied; or, one could write "often, it is practical to have many more greenlets than it is practical to have threads."

Both of those variations are extremely wordy, so "it" is used to make the sentence flow better. True, "it" is asked to do a lot of heavy lifting, and in speaking, one would tend to emphasize that "it" a bit to indicate this (try reading with a slight pause after the "it").

Even though I studied both German and Latin, there's no way I would try to write anything in those languages today so I really appreciate your inputs. I'll try to keep this discussion in mind for future revisions.